### PR TITLE
Complete As an IT manager, I want to manage user acconuts, so that I …

### LINKED ISSUE #195: Create a web page to display users’ account information

### LINKED ISSUE #196: Create an API to fetch users’ information

### LINKED ISSUE #197: Connect web page to API

### PR DESCRIPTION
**Pull request by shuang81**
Close #195 
Close #196 
Close #197

### CODE REVIEW

**Comment by shuang81** [comment]
Resolve the conflicts

File: backend/controllers/profile.controller.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ const User = require('../models/user.model.js');
 
 exports.getUserProfile = async (req, res) => {
   try {
-    const user = await User.findById(res.locals.userId).select('-password');
```
Comment (paneleon) [inline backend/controllers/profile.controller.js:5]: can you please keep the "res.locals.userId", it's already implemented and it's working. The value comes from authentication middleware

File: backend/controllers/user.controller.js [hunk 2]
```diff
@@ -1,79 +1,10 @@
 const passport = require('passport');
 const authUtils = require('../utils/auth.js');
-const User = require('../models/user.model')
```
Comment (paneleon) [inline backend/controllers/user.controller.js:3]: please restore the functions you deleted from this file, this is breaking the application.

File: backend/routes/emergency.route.js [hunk 3]
```diff
@@ -3,8 +3,12 @@ const router = express.Router();
 const authUtils = require('../utils/auth.js');
 const EmergencyController = require('../controllers/emergency.controller.js');
 
-router.get('/', authUtils.isAuthenticated, EmergencyController.getUserEmergency);
+
+router.get('/', authUtils.auth, EmergencyController.getUserEmergency);
```
Comment (paneleon) [inline backend/routes/emergency.route.js:7]: the function is called isAuthenticated, not auth, change it back to authUtils.isAuthenticated pls.
.

File: backend/routes/manage.route.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const router = express.Router();
+const authUtils = require('../utils/auth.js');
+const ManageController = require('../controllers/manage.controller.js');
+
+
+router.get('/', authUtils.auth, ManageController.getUserData);
```
Comment (paneleon) [inline backend/routes/manage.route.js:7]: authUtils.isAuthenticated

File: backend/routes/payment.route.js [hunk 5]
```diff
@@ -3,7 +3,10 @@ const router = express.Router();
 const authUtils = require('../utils/auth.js');
 const PaymentController = require('../controllers/payment.controller.js');
 
-router.get('/', authUtils.isAuthenticated, PaymentController.getUserPayment);
-router.put('/', authUtils.isAuthenticated, PaymentController.updatePayment);
+
+router.get('/', authUtils.auth, PaymentController.getUserPayment);
```
Comment (paneleon) [inline backend/routes/payment.route.js:7]: authUtils.isAuthenticated

File: backend/routes/profile.route.js [hunk 6]
```diff
@@ -3,8 +3,8 @@ const router = express.Router();
 const authUtils = require('../utils/auth.js');
 const ProfileController = require('../controllers/profile.controller.js');
 
-router.get('/', authUtils.isAuthenticated, ProfileController.getUserProfile);
-router.put('/', authUtils.isAuthenticated, ProfileController.updateProfile);
+router.get('/', authUtils.auth, ProfileController.getUserProfile);
```
Comment (paneleon) [inline backend/routes/profile.route.js:6]: authUtils.isAuthenticated

File: backend/routes/status.route.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+const authUtils = require('../utils/auth.js');
+const StatusController = require('../controllers/status.controller.js');
+
+
+router.get('/', authUtils.auth, StatusController.getStatus);
```
Comment (paneleon) [inline backend/routes/status.route.js:7]: authUtils.isAuthenticated

File: backend/routes/subscription.route.js [hunk 8]
```diff
@@ -3,7 +3,7 @@ const router = express.Router();
 const authUtils = require('../utils/auth.js');
 const SubscriptionController = require('../controllers/subscription.controller.js');
 
-router.get('/', authUtils.isAuthenticated, SubscriptionController.getSubscription);
-router.put('/', authUtils.isAuthenticated, SubscriptionController.updateSubscription);
+router.get('/', authUtils.auth, SubscriptionController.getSubscription);
```
Comment (paneleon) [inline backend/routes/subscription.route.js:6]: authUtils.isAuthenticated

File: backend/routes/user.route.js [hunk 9]
```diff
@@ -3,9 +3,9 @@ const UserController = require('../controllers/user.controller.js');
 const authUtils = require('../utils/auth.js')
 const router = express.Router();
 
-router.post('/login', UserController.processLogin);
```
Comment (paneleon) [inline backend/routes/user.route.js:6]: please restore already working functionality.

**Review by paneleon — CHANGES_REQUESTED** [review]

**Pull request merged**